### DISCUSSION TITLE
How To Change The Cursor?

### DISCUSSION THREAD

**Discussion by macaluzate** [JaKooLit/Hyprland-Dots#481] [Q&A] [answered]
Hello, i was trying to change it from initial-boot.sh but i think i'm doing it in the wrong way, the i tried to paste  the cursor folder in ~/.icons and change the /usr/share/icons/default/index.theme but when i reboot, now appears the hypr cursor, idk why. 

i appreciate the help.

**Comment by JaKooLit** [comment, marked as answer]
use nwg-look instead

**Comment by macaluzate** [comment]
Thank you , i'm noticing  the cursor is different  over some apps , is there any way to change it entirely?

**Reply by JaKooLit** [reply]
I dont know what changes you've made so I cant tell

https://youtu.be/nJLnRgnLPWI?si=8W3e-nJaqayWyZH2

11:12

**Reply by macaluzate** [reply]
niceee, i just re did the default config, and rebooted my machine, that was it.
